database/seeds/testing/24_test_visit.seed.js

```diff
@@ -1 +1 @@
-module.exports = require('../test_data/7_test_visit.seed');
```

Comment:
this file needs renaming?

Reply:
It has been renamed. See above diff.

Reply:
Can't see `database/seeds/testing/24_test_visit.seed.js` -> `database/seeds/testing/24_test_visit_logs.seed.js`

Reply:
Thought you meant the file mentioned in your comment, rather than the file commented on. Changed in amend